S2VX.Game.Tests/HeadlessTests/ScoreProcessorTests/ScoreStatisticsTests.cs

```diff
@@ -0,0 +1,141 @@
+﻿using NUnit.Framework;
+using osu.Framework.Allocation;
+using osu.Framework.Testing;
+using S2VX.Game.Play.UserInterface;
+using S2VX.Game.Story;
+using S2VX.Game.Story.Note;
+
+namespace S2VX.Game.Tests.HeadlessTests.ScoreProcessorTests {
+    [HeadlessTest]
+    public class ScoreStatisticsTests : S2VXTestScene {
+        [Cached]
+        private S2VXStory Story { get; } = new();
+
+        private Notes Notes { get; set; }
```

Comment:
Is it possible to initialize this as = Story.Notes here so you don't have to do it in the set up steps? Might be applicable to other files in this change if this works.

Reply:
No, this is not possible. You can't set properties based on other properties because the order of properties in a class is not guaranteed. Think of this possible scenario that wouldn't make sense:
```csharp
class Example {
    string a = new string(b);
    string b = new string(a);
}
```

Reply:
Oh aite